sdk/search/search-documents/src/generated/service/models/index.ts

```diff
@@ -4104,6 +4126,59 @@ export interface IndexesAnalyzeOptionalParams
 /** Contains response data for the analyze operation. */
 export type IndexesAnalyzeResponse = AnalyzeResult;
 
+/** Optional parameters. */
+export interface AliasesCreateOptionalParams
+  extends coreClient.OperationOptions {
+  /** Parameter group */
+  requestOptionsParam?: RequestOptions;
+}
+
+/** Contains response data for the create operation. */
+export type AliasesCreateResponse = SearchAlias;
+
+/** Optional parameters. */
+export interface AliasesListOptionalParams extends coreClient.OperationOptions {
+  /** Parameter group */
+  requestOptionsParam?: RequestOptions;
+}
+
+/** Contains response data for the list operation. */
+export type AliasesListResponse = ListAliasesResult;
+
+/** Optional parameters. */
+export interface AliasesCreateOrUpdateOptionalParams
+  extends coreClient.OperationOptions {
+  /** Parameter group */
+  requestOptionsParam?: RequestOptions;
+  /** Defines the If-Match condition. The operation will be performed only if the ETag on the server matches this value. */
+  ifMatch?: string;
+  /** Defines the If-None-Match condition. The operation will be performed only if the ETag on the server does not match this value. */
+  ifNoneMatch?: string;
```

Comment:
It seems like we are not giving our users the option to set this option. Is that expected?